formbricks/formbricks
# [DRAFT] feat(packages/surveys): ability to customize colors & other improvements

## What does this PR do?

This PR adds the ability to deeply customize all `packages/surveys` components using CSS variables. It also removes the `brandColor` property from the components.

Fixes # (issue)



## Type of change




## How should this be tested?



- Check that the default theme works
- Check that it can be customized using CSS Variables.

## Checklist
